image.go

```diff
@@ -0,0 +1,7 @@
+package spotify
+
+type Image struct {
+	Height *int
```

Comment:
As far as I know, there's no benefit to making `int`s pointers

Reply:
I was finding there were cases where the API did not send these fields along, so I figured it might be useful to have a nil-check-able field. I can make these non-pointer, and I suppose a zero-check would work for those cases.